src/Scripts/EventBus.gd

```diff
@@ -18,6 +18,9 @@ signal add_healthbar(healthbar)
 signal remove_healthbar(healthbar)
 signal activate_tile(tile)
 
+signal create_explosion(target, damage, radius)
+
 # skills
```

Comment:
These could do with a rename eg. `stun_pistol` to `fired_stun_pistol` or something.

Reply:
While this may make more sense, it violates the unwritten convention of having signals for skills defined as just the skill name.